source/style-guide/markup/directives/sharedinclude.txt

```diff
@@ -0,0 +1,234 @@
+.. _style-guide-sharedinclude:
+
+==============
+Shared Include
+==============
+
+The ``sharedinclude`` directive allows you to source content from a file
+located in the MongoDB internal `docs-shared <https://github.com/10gen/docs-shared>`__
+repository. Use this directive when you need to keep the same content in sync
+for multiple sets of documentation or multiple branches of a documentation set.
+
+This page uses the following terms:
+
+- **External file**: content committed to the ``docs-shared`` repository
+  that can be sourced by other documentation repositories.
+- **Sourcing file**: content in a documentation repository that uses a
+  ``sharedinclude`` directive to pull content from an external file.
+- **Placeholder**: a variable that indicates text that the sourcing file
+  must provide a replacement for. These are located in external files and are
+  represented by the placeholder name surrounded by the ``|`` (pipe)
+  character.
+
+.. tip::
+
+   The ``sharedinclude`` directive is an extension of the ``include`` directive.
+   While the ``include`` directive allows you to source content from the
+   branch of the repository in which it is used, the ``sharedinclude``
+   directive is hardcoded to reference external files from the
+   `10gen/docs-shared <https://github.com/10gen/docs-shared/>`__
+   private GitHub repository.
+
+
+Setup
+-----
+
+You must add the URL from which to source the shared content in the
+``sharedinclude_root`` setting of the documentation repository's ``snooty.toml``
+configuration file. This should reference the unprocessed versions of the files,
+currently served from the ``raw.githubusercontent.com`` domain.
+
+If the shared content that your docs repository file references is on the
+``main`` branch of ``docs-snooty``, your ``snooty.toml`` configuration entry
+should resemble the following line:
```

Comment:
```suggestion
For example, if the shared content that your docs repository file references is on the
``main`` branch of ``docs-mongodb-internal``, your ``snooty.toml`` configuration entry
should resemble the following line:
```
Is this supposed to be an example or is ``docs-snooty`` a real repo? If example, changed the example repo name to be less confusing with ``snooty.toml``. If it is a real repo, disregard!

Reply:
Whoops, good catch. This should be docs-shared to correspond with the repo referenced in the code-block.